File: 5.22/门店会员综合管理系统/server/middlewares/response.js
const responseHandler = (req, res, next) => {
  res.success = (data = null, message = 'success') => {
    res.json({
      code: 200,
      message,
      data
    })
  }

  res.error = (message = 'error', code = 400, data = null) => {
    res.json({
      code,
      message,
      data
    })
  }

  next()
}

module.exports = responseHandler

File: 5.22/企业月度营收数据分析平台/app/utils/response.py
from flask import jsonify

def success_response(data=None, message='success', code=200):
    response = {
        'code': code,
        'message': message,
        'data': data
    }
    return jsonify(response), code

def error_response(message='error', code=500, data=None):
    response = {
        'code': code,
        'message': message,
        'data': data
    }
    return jsonify(response), code
